tests/lib.rs

```diff
@@ -110,6 +110,26 @@ fn connection_set_busy_handler() {
     }
 }
 
+#[test]
+fn connection_select() {
```

Comment:
I guess this will go away, and `Cursor`’s tests and examples will be eventually updated.

Reply:
Indeed, I removed it.

Though I renamed the existing `next()` function to `try_next()` and I added a test for this in 09336ef